fjord/feedback/jinja2/feedback/android_feedback.html

```diff
@@ -0,0 +1,199 @@
+                    </div>{# .message #}
+                    <div>
+                      {% trans %}
+                        It takes less than a minute and feels great.
+                      {% endtrans %}
+                    </div>
+                    <a class="stars" id="play-store" data-event="FeedbackOpenPlay" href="#">
+                      <svg>
+                        <defs>
+                          <symbol viewBox="0 0 100 100" id="star">
+                            <polygon points="48 75 18.6107374 90.4508497 24.2235871 57.7254249 0.447174185 34.5491503 33.3053687 29.7745751 48 0 62.6946313 29.7745751 95.5528258 34.5491503 71.7764129 57.7254249 77.3892626 90.4508497" class="star" xmlns="http://www.w3.org/2000/svg" fill="#0092db"></polygon>
+                          </symbol>
+                        </defs>
+                        <use width="25" height="25" xlink:href="#star" x="0" y="0"></use>
+                        <use width="25" height="25" xlink:href="#star" x="25" y="0"></use>
+                        <use width="25" height="25" xlink:href="#star" x="50" y="0"></use>
+                        <use width="25" height="25" xlink:href="#star" x="75" y="0"></use>
+                        <use width="25" height="25" xlink:href="#star" x="100" y="0"></use>
+                      </svg>
+                      {% trans %}
+                        Yes, go to Google Play
+                      {% endtrans %}
+                    </a>
+                  </div>{# .message-box #}
+                </div>{# .happy #}
+
+                <div class="sad">
+                  <h1>
+                    {% trans %}
+                      Oh no!
+                    {% endtrans %}
+                  </h1>
+
+                  <div class="ask">
+                    <form id="responseform" action="" method="post">
+                      <div class="input-box">
+                        <label for="description">
+                          {% trans %}
+                            We're sorry that you had some problems with Firefox.
+                            Please tell us what happened so that we can fix it.
+                          {% endtrans %}
+                        </label>
+                        <div id="description-counter">{{ TRUNCATE_LENGTH }}</div>
+                        <textarea data-max-length="{{ TRUNCATE_LENGTH }}" id="description" name="description" placeholder="Enter your feedback here"></textarea>
+                      </div>{# .input-box #}
+
+                      <div class="input-box">
+                        <label for="id_url">
+                          {% trans %}
+                            Include last visited site
+                          {% endtrans %}
+                          <input id="last-checkbox" type="checkbox" checked="checked"/>
+                          <input class="url" id="id_url" name="url" placeholder="http://" type="text" readonly="readonly">
```

Comment:
I filed a bug to get this from the url param https://bugzilla.mozilla.org/show_bug.cgi?id=1239513